src/test/java/org/broadinstitute/consent/http/resources/DatasetResourceTest.java

```diff
@@ -366,50 +386,56 @@ void testValidateDatasetNameSuccess() {
     Dataset testDataset = new Dataset();
     when(datasetService.getDatasetByName("test")).thenReturn(testDataset);
     initResource();
-    Response response = resource.validateDatasetName("test");
-    assertEquals(HttpStatusCodes.STATUS_CODE_OK, response.getStatus());
+    try(var response = resource.validateDatasetName("test")) {
```

Comment:
whitespace
```suggestion
    try (var response = resource.validateDatasetName("test")) {
```

Is this change to satisfy a lint warning?